apps/ex_marketer/test/ex_marketer/crawler/task_supervisor_test.exs

```diff
@@ -1,75 +1,62 @@
 defmodule ExMarketer.Crawler.TaskSupervisorTest do
   use ExMarketer.DataCase, async: true
-  use ExVCR.Mock, adapter: ExVCR.Adapter.Hackney
 
   alias ExMarketer.Crawler.TaskSupervisor
   alias ExMarketer.Keyword
 
   describe "given a successful response" do
     @tag :skip
     test 'start_chilld/1 spawn a new process' do
-      use_cassette "google/valid" do
-        {:ok, pid} = TaskSupervisor.start_chilld("grammarly", insert(:user).id)
+      {:ok, pid} = TaskSupervisor.start_chilld("grammarly", insert(:user).id)
```

Comment:
You have had this typo for a while 😁 

```suggestion
      {:ok, pid} = TaskSupervisor.start_child("grammarly", insert(:user).id)
```

Reply:
Wow, thanks for pointing it out @olivierobert 😂 honestly I don't check the typo much :D I think I should use some Grammar checking extensions on VSCode now, on Ruby Mine it has as default so it's easy to detect the typo there 😂